azure-devops/global/stages/50-deployment/deploy-az-function.yaml

```diff
@@ -22,6 +22,11 @@ steps:
     displayName: 'Read Outputs from file'
 
   - template: './database.yaml'
+    parameters:
+      PIPELINE_FIREWALL_NAME: $(PIPELINE_FIREWALL_NAME)
```

Comment:
How are you passing these parameters if you are not receiving? You keeped it implicit, that's why it might work, but now it's confusing. It could be an environment variable or a parameter, we don't know just by reading. And it's a consistency issue.